Products/CMFPlomino/fields/datetime.py

```diff
@@ -148,6 +159,11 @@ def processInput(self, submittedValue):
                 return StringToDate(submittedValue, fmt)
         # it is instance type when no js is detected
         # submittedValue = ampm: , day: 09, hour: 00, minute: 00, month: 03, year: 1993
+        if submittedValue.year == '0000' or \
+           submittedValue.month == '00' or \
+           submittedValue.day == '00':
+            return None
+
```

Comment:
if one has a valid value but the others don't. Shouldn't that be a validation issue rather than the same as reseting the value?